lively.merger/merger.js

```diff
@@ -160,26 +235,26 @@ export async function mergeMorphsWithIds (
   if (!morphB) {
     throw new Error(`Cannot merge morphs, morphB with id ${morphBid} not found`);
   }
-  return mergeMorphs(
-    morphA, 
-    morphB, 
-    onMergeResult);
+  return await mergeMorphs(
```

Comment:
Return await is an anti-pattern because it creates a new promise which just awaits for mergeMorphs' promise to be fulfilled. Just use return mergeMorphs(...) instead

```suggestion
  return mergeMorphs(
```